src/main/java/com/diamondfire/helpbot/bot/command/impl/other/fun/IdeaCommand.java

```diff
@@ -33,198 +35,347 @@ public Permission getPermission() {
-                "for money.",
-                "for a ban from DiamondFire.",
-                "for a big prize.",
-                "for an exclusive BuzzFeed article.",
-                "for Develoepr.",
-                "with a turtle.",
-                "with friends.",
-                "with a very big elephant.",
-                "for a small compliment.",
-                "for a troll.",
-                "for the fall of death.",
-                "for a visit to prison.",
-                "to visit McDonalds.",
-                "for a big McChicken burger.",
-                "for an egg from a chicken.",
-                "for another plot.",
-                "for a neat firework show.",
-                "to listen to Pigstep.",
-                "for a show from DJ RyanLand.",
-                "for a bunch of ducks saying quack.",
-                "for good game ideas.",
-                "for a volcano eruption.",
-                "for the new DiamondFire update.",
-                "to get banned.",
-                "for samquotes.",
-                "for love.",
-                "for Robot Game levels.",
-                "for Hypercube source code.",
-                "for warm blankets.",
-                "for the labsCore API.",
-                "for V-Bucks.",
-                "to get hugs from Owen.",
-                "to eat food.",
-                "for red pandas",
+        
+        String[] themes = new String[] {
+                "Ancient",
+                "Automatic",
+                "Broken",
+                "Cavernous",
+                "City",
+                "Construct",
+                "Dark",
+                "Decay",
+                "Deceptive",
+                "Deleted",
+                "Depths",
+                "Desert",
+                "Dimensional",
+                "Fantasy",
+                "Freedom",
+                "Futuristic",
+                "Gigantic",
+                "Glitchy",
+                "Growth",
+                "Isolated",
+                "Laboratory",
+                "Linked",
+                "Lush",
+                "Magical",
+                "Mirrored",
+                "Modern",
+                "Monochromatic",
+                "Mythical",
+                "Night",
+                "Plain",
+                "Power",
+                "Remote",
+                "Ruined",
+                "Science",
+                "Shattered",
+                "Simplistic",
+                "Snowy",
+                "Space",
+                "Spy",
+                "Surreal",
+                "Swarm",
+                "Torment",
+                "Underground",
+                "Underwater",
+                "Vanish",
+                "Zombie"
         };
         
-        Random rdm = new Random();
-        EmbedBuilder builder = new EmbedBuilder();
+        Random rng = new Random();
+        
+        int[] values = new int[]{1,2,3,4};
+        int[] weights = new int[]{10,5,2,1};
```

Comment:
But generally if possible I would store these in field too. Essentially any value that is a constant can be reused.